src/include/miopen/mlo_internal.hpp

```diff
@@ -151,8 +156,11 @@ class StaticContainer
         return data;
     }
 };
-
-#if MIOPEN_ENABLE_SQLITE
+#if MIOPEN_LINK_DB && MIOPEN_ENABLE_SQLITE
+using PerformanceDb = DbTimer<MultiFileDb<PerfRamDb, SQLitePerfDb, true>>;
+#elif MIOPEN_LINK_DB
```

Comment:
Embedding perf-db in text format is not supported. Please remove.

Reply:
Fixed